### DISCUSSION TITLE
No updated documentation and examples

### DISCUSSION THREAD

**Discussion by antillgrp** [Kitware/trame#304] [Trame/VTK] [answered]
I there any way to get documentation for the latest version, it seems that the API has changed so some of the examples are not able to run properly , for example:

(.venv) PS C:\Users\G\Trame\trame-1> python.exe .\examples\02_vuetify\02_router.py --server

Error: 

ImportError: cannot import name 'RouterViewLayout' from 'trame' (C:\Users\G\Trame\trame-1\.venv\lib\site-packages\trame\__init__.py)

Versions
    [latest](https://trame.readthedocs.io/en/latest/trame.widgets.trame.html) 
    [v1.20.0](https://trame.readthedocs.io/en/v1.20.0/trame.widgets.trame.html) 

On Read the Docs

**Comment by jourdain** [comment]
So the issue you are running into is related to trame v3. You should see a message before that import error telling you that you need to pip install the widgets that your app is using. Basically we removed the explicit dependencies to external widgets by default to reduce trame size to a minimum. 

I guess, I should add requirements.txt across the examples to overcome such issue.

In can see the list of widgets available [here](https://kitware.github.io/trame/docs/trame_v2-3.html) with their corresponding package name to pip install.

Based on those imports

```python
from trame.ui.vuetify import SinglePageWithDrawerLayout
from trame.ui.router import RouterViewLayout
from trame.widgets import vuetify, router
```

You will need to `pip install trame trame-vuetify trame-router`

**Comment by jourdain** [comment, marked as answer]
The examples have be updated either by updating the `requirements.txt` to add the missing `trame-*` or with a comment on the top to list the expected `pip install ...` command that is expected to run such example.

Hopefully that should prevent the issue you ran into. Thanks for reporting your finding.
